src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/UserCreationRequest.java

```diff
@@ -33,7 +33,7 @@ public UserCreationRequest(
         this.firstName = firstName;
         this.lastName = lastName;
 
-        if (!StringUtils.isEmpty(emailAddress)) {
+        if (StringUtils.hasLength(emailAddress)) {
```

Comment:
can we use isBlank from apache lib ? checking length this looks misleading

Reply:
I can change, although its the same thing

Reply:
please check the blog 
https://github.com/spring-projects/spring-framework/issues/25945

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1038/commits/e66d205e7a4c2e85253e9295731dc1d334e5d94a